### ISSUE TITLE
Publish 0.14 alphas to npm

### ISSUE THREAD

**Issue by dantman** [facebook/react#3850]
The `react@next` mentioned in #2924 currently points to an old 0.13.0 rc.

It would be nice to have 0.14 alphas published since some things like context coming from parents as I understand are post 0.13 and one can't just `npm install` the git repo #master to test if that fixes things.

**Comment by andreypopp** [comment]
There are nightly builds published at http://react.zpao.com/builds/ (are they being published nightly, @zpao)?

Probably it makes sense to document that fact somewhere.

**Comment by cody** [comment]
There is also a fiddle with the nightly build: https://jsfiddle.net/reactjs/ghmpo42k/

**Comment by zpao** [comment]
@dantman Thanks for bringing this up, we've been bad about this. Our intention was to publish the same revs that we sync internally @ FB but then we didn't do that. We just synced recently so I published what's in master as `0.14.0-alpha1`

@andreypopp They're being published on each commit (at least the ones that we didn't decide are docs only). @spicyj and I were just talking Friday about moving the builds off my website & into a more official home.

@cody not sure we ever shared it but nice detective work :)
